VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/non-host-waiting-room/non-host-waiting-room.component.html

```diff
@@ -0,0 +1,207 @@
+<div *ngIf="loadingData">
+  <p class="govuk-body">{{ 'waiting-room.loading-please-wait' | translate }}</p>
+</div>
+
+<app-video-call
+  *ngIf="showVideo"
+  id="video-container"
+  [isJohRoom]="isJohRoom"
+  [canToggleParticipantsPanel]="isPrivateConsultation"
+  [isPrivateConsultation]="isPrivateConsultation"
+  [vhParticipant]="vhParticipant"
+  [hasCaseNameOverflowed]="hasCaseNameOverflowed"
+  [vhConference]="vhConference"
+  [connected]="connected"
+  [outgoingStream]="outgoingStream"
+  [isSupportedBrowserForNetworkHealth]="isSupportedBrowserForNetworkHealth"
+  [showConsultationControls]="showConsultationControls"
+  [isParticipantsPanelHidden]="isParticipantsPanelHidden"
+  [hearing]="hearing"
+  [participantEndpoints]="participantEndpoints"
+  [showVideo]="showVideo"
+  [presentationStream]="presentationStream"
+  [streamInMain]="streamInMain"
+  [callStream]="callStream"
+  [roomName]="getRoomName()"
+  [caseNameAndNumber]="getCaseNameAndNumber()"
+  [isParticipantsPanelEnabled]="isPrivateConsultation"
+  (ready)="setTrapFocus()"
+  (leaveConsultation)="showLeaveConsultationModal()"
+  (consultationLockToggle)="setRoomLock($event)"
+  (deviceToggle)="showChooseCameraDialog()"
+  (languageChange)="showLanguageChangeModal()"
+  (participantsPanelToggle)="toggleParticipantsPanel()"
+  (feedToggle)="switchStreamWindows()"
+>
+</app-video-call>
+
+<app-transfer-message *ngIf="isParticipant && isTransferringIn" [outgoingStream]="outgoingStream"></app-transfer-message>
```

Comment:
I think the transfer message should appear for all participants being invited in for a consistent experience else judicial users have a blank screen whereas non judicial get a clear message